lib/improver/tests/acceptance/test_blend_adjacent_points.py

```diff
@@ -92,9 +100,10 @@ def test_time(tmp_path):
     kgo_dir = acc.kgo_root() / "blend-adjacent-points/time_bounds"
     multi_prob = sorted(kgo_dir.glob("*wind_gust*.nc"))
     output_path = tmp_path / "output.nc"
-    args = ["time", "1536908400",
+    args = ["--coordinate", "time", "--central-point", "1536908400",
```

Comment:
Wait, what?

Reply:
No idea - I just took what was already there!

Reply:
Fair point.
Thinking about it I think this is there just to test `"time"` throws a helpful error (since we cannot handle it within the code, let alone such an interface being sensible!).  So the values are irrelevant details.

Reply:
Agreed, we should probably put something that's more obviously wrong like `"badtime"`

Reply:
(Changing the value to a bad time caused a different error, so left as-is (with something numeric) - we could test the other error, but I don't think it's important really).